src/app/plan/duck/operations.ts

```diff
@@ -126,9 +129,19 @@ const addPlan = migPlan => {
 
       dispatch(addPlanSuccess(createRes.data));
 
-      console.debug('Beginning PV polling');
-
+      let timesRun = 0;
       const interval = setInterval(async () => {
+        dispatch(pvFetchRequest());
+        timesRun += 1;
+        // TODO: replace timesRun with poller class
+        if (timesRun === 3) {
```

Comment:
I don't remember what the time interval is, but we should bump this up so that it's a total of something like 30 seconds. I've seen it take on the order of 10. Also, we should put the times run condition in a const var that's defined next to the polling interval var, so that is easy to tweak:

```
const watchInterval = 5000; // 5 seconds
const watchTicks = 12; // Check 12 times, so timeout is 60 secs

//...
         if (timesRun === watchTicks) {
```

Reply:
Updating to use timesRun === 12. The times run is set outside the interval function using let so that it can be updated on every tick of the interval. Definitely think this should be scrapped asap for the poller class though.